src/fidesops/api/v1/exception_handlers.py

```diff
@@ -0,0 +1,21 @@
+from typing import Callable, List
+
+from fastapi import Request
+from fastapi.responses import JSONResponse, Response
+from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR
+
+from fidesops.common_exceptions import FunctionalityNotConfigured
+
+
+class ExceptionHandlers:
+    @staticmethod
+    def functionality_not_configured_handler(
+        request: Request, exc: FunctionalityNotConfigured
+    ) -> JSONResponse:
+        return JSONResponse(
+            status_code=HTTP_500_INTERNAL_SERVER_ERROR, content={"message": str(exc)}
```

Comment:
@sanders41  while you're here - any thoughts on `500` as the error code here? i considered other options but anything else i could think of seemed like it could be misinterpreted, whereas i feel like a generic `500` with a clear error message probably has the best chance of being well understood. but i'd like a second opinion on this!

Reply:
`500` make sense to me here. The other option I could see making sense is `503`, but `503` I think would imply things are down temporarily and will come back up which won't be the case so that could be confusing.